tools/benchmark

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+LEGOFY=$(pwd)/bin/legofy;
+PATH_IMAGES=$(pwd)/images/benchmark/;
+
+benchmark() {
+  local flags=$@;
+
+  # Default arguments
+  if [[  $flags = '' ]]; then
+    flags='--filter-quality=80 --pattern-quality=80 --quality=good'
```

Comment:
Nothing in da [readme](https://github.com/HugoGiraudel/node-legofy) except for quality :/